src/Admin/Vue.php

```diff
@@ -459,6 +459,7 @@ private function payplug_requirements() {
 	public function payplug_section_help( ) {
 
 		$help = [
+			"save_changes_text" => __("payplug_save_changes_text", "payplug"),
 			"description1" => __("payplug_section_help_description1", "payplug"),
```

Comment:
Please use significant naming, instead of description1 and description2 you can use only one "description" that is composed of 2 elemtents

Reply:
Done